ui/app/AppLayouts/Chat/panels/communities/CommunityMintTokensSettingsPanel.qml

```diff
@@ -244,4 +269,28 @@ SettingsPageLayout {
             }
         }
     }
+
+    Component {
+        id: signTransactionDialog
+
+        SignMintTokenTransactionPopup {
+            id: popup
+
+            onOpened: root.signMintTransactionOpened()
+            onCancelClicked: close()
+            onSignTransactionClicked: root.signMintTransactionClicked()
+
+            Binding {
```

Comment:
is it necessary to use `Binding` object instead of regular binding?

Reply:
Since this values are from `root` and can be change any time, by using regular binding in lines `203` and `204` like this:
```
feeText: Qt.binding(()=> root.feeText),
isFeeLoading: Qt.binding(()=> root.isFeeLoading),
```
when the component is destroyed, but these values are changed, that triggers qml warnings, since the component is `undefined`.

Reply:
just
```
feeText: root.feeText,
isFeeLoading: root.isFeeLoading,
```
also generates warnings?

Reply:
ok, I see, it's related to Global.openPopup that creates popup and holds the instance forever. The solution is to keep the popup local. Than it's not kept unnecessarily in memory, code is simpler, no dependency on global object and everything working in storybook :) :
```
    Component {
        id: collectibleView

        CommunityCollectibleView {
            viewWidth: root.viewWidth
            preview: d.preview
            holdersModel: root.holdersModel

            onMintCollectible: {
                d.collectibleName = name
                popup.open()
            }

            Connections {
                target: root

                function onSignMintTransactionClicked() {
                    root.isFeeLoading = true
                    root.feeText = ""
                    root.mintCollectible(artworkSource,
                                         name,
                                         symbol,
                                         description,
                                         parseInt(supplyText),
                                         infiniteSupply,
                                         transferable,
                                         selfDestruct,
                                         chainId,
                                         accountName,
                                         d.accountAddress)

                    stackManager.clear(d.initialViewState, StackView.Immediate)
                }
            }

            SignMintTokenTransactionPopup {
                id: popup

                collectibleName: parent.name
                accountName: parent.accountName
                networkName: parent.chainName
                feeText: root.feeText
                isFeeLoading: root.isFeeLoading

                onOpened: root.signMintTransactionOpened()
                onCancelClicked: close()
                onSignTransactionClicked: root.signMintTransactionClicked()
            }
        }
    }
```